### PR TITLE
Add NPCs from PFS Quest 14

### PR DESCRIPTION
**Pull request by TMunsie**
Also update license file for Q14

### CODE REVIEW

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json [hunk 1]
```diff
@@ -0,0 +1,1992 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "2d6"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>Gouts of flame rush from your hands. You deal 2d6 fire damage to creatures in the area.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The damage increases by 2d6.</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json:478]: Should the description be updated?

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json [hunk 2]
```diff
@@ -0,0 +1,1992 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "1d4"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>A small ball of flame appears in the palm of your hand, and you lash out with it either in melee or at range. Make a spell attack roll against your target's AC. This is normally a ranged attack, but you can also make a melee attack against a creature in your unarmed reach.</p>\n<p>On a success, you deal 1d4 fire damage plus your spellcasting ability modifier. On a critical success, the target takes double damage and 1d4 persistent fire damage.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Increase the damage by 1d4 and the persistent damage on a critical hit by 1d4.</p>\n<p>[[/r (@item.level)d4[persistent,fire]]]{Leveled Persistent Fire Damage}</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json:1166]: Should the description an the leveled persistent damage link be updated?

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json [hunk 3]
```diff
@@ -0,0 +1,1992 @@
...
+        },
+        {
+            "_id": "1EsMXmB9tTK4sElQ",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json:1743]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-3-4.json [hunk 4]
```diff
@@ -0,0 +1,2737 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "2d6"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>Gouts of flame rush from your hands. You deal 2d6 fire damage to creatures in the area.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The damage increases by 2d6.</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-rhenei-3-4.json:879]: Description?

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-3-4.json [hunk 5]
```diff
@@ -0,0 +1,2737 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "1d4"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>A small ball of flame appears in the palm of your hand, and you lash out with it either in melee or at range. Make a spell attack roll against your target's AC. This is normally a ranged attack, but you can also make a melee attack against a creature in your unarmed reach.</p>\n<p>On a success, you deal 1d4 fire damage plus your spellcasting ability modifier. On a critical success, the target takes double damage and 1d4 persistent fire damage.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Increase the damage by 1d4 and the persistent damage on a critical hit by 1d4.</p>\n<p>[[/r (@item.level)d4[persistent,fire]]]{Leveled Persistent Fire Damage}</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-rhenei-3-4.json:1911]: Description and persistent damage link?

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-3-4.json [hunk 6]
```diff
@@ -0,0 +1,2737 @@
...
+        },
+        {
+            "_id": "1EsMXmB9tTK4sElQ",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-rhenei-3-4.json:2488]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/determined-thisnabel-1-2.json [hunk 7]
```diff
@@ -0,0 +1,625 @@
...
+        },
+        {
+            "_id": "rm5sOGU46Lbwed4O",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-thisnabel-1-2.json:319]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/determined-thisnabel-1-2.json [hunk 8]
```diff
@@ -0,0 +1,625 @@
...
+                },
+                "description": {
+                    "value": "<p>Thisnabel deals an extra 1d4 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "category": "precision",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-thisnabel-1-2.json:395]: Should this use the full sneak attack RE?

File: packs/data/pfs-season-4-bestiary.db/determined-thisnabel-3-4.json [hunk 9]
```diff
@@ -0,0 +1,625 @@
...
+        },
+        {
+            "_id": "rm5sOGU46Lbwed4O",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-thisnabel-3-4.json:319]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/determined-thisnabel-3-4.json [hunk 10]
```diff
@@ -0,0 +1,625 @@
...
+                },
+                "description": {
+                    "value": "<p>Thisnabel deals an extra 1d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "category": "precision",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/determined-thisnabel-3-4.json:395]: Full sneak attack RE?

File: packs/data/pfs-season-4-bestiary.db/rhenei-1-2.json [hunk 11]
```diff
@@ -0,0 +1,1528 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "1d4"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>A small ball of flame appears in the palm of your hand, and you lash out with it either in melee or at range. Make a spell attack roll against your target's AC. This is normally a ranged attack, but you can also make a melee attack against a creature in your unarmed reach.</p>\n<p>On a success, you deal 1d4 fire damage plus your spellcasting ability modifier. On a critical success, the target takes double damage and 1d4 persistent fire damage.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Increase the damage by 1d4 and the persistent damage on a critical hit by 1d4.</p>\n<p>[[/r (@item.level)d4[persistent,fire]]]{Leveled Persistent Fire Damage}</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/rhenei-1-2.json:702]: Description and level damage link

File: packs/data/pfs-season-4-bestiary.db/rhenei-1-2.json [hunk 12]
```diff
@@ -0,0 +1,1528 @@
...
+        },
+        {
+            "_id": "1EsMXmB9tTK4sElQ",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/rhenei-1-2.json:1279]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/rhenei-3-4.json [hunk 13]
```diff
@@ -0,0 +1,2114 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "2d6"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>Gouts of flame rush from your hands. You deal 2d6 fire damage to creatures in the area.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The damage increases by 2d6.</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/rhenei-3-4.json:478]: Description?

File: packs/data/pfs-season-4-bestiary.db/rhenei-3-4.json [hunk 14]
```diff
@@ -0,0 +1,2114 @@
...
+                                "categories": [],
+                                "subtype": "",
+                                "value": "bludgeoning"
+                            },
+                            "value": "1d4"
+                        }
+                    }
+                },
+                "description": {
+                    "value": "<p>A small ball of flame appears in the palm of your hand, and you lash out with it either in melee or at range. Make a spell attack roll against your target's AC. This is normally a ranged attack, but you can also make a melee attack against a creature in your unarmed reach.</p>\n<p>On a success, you deal 1d4 fire damage plus your spellcasting ability modifier. On a critical success, the target takes double damage and 1d4 persistent fire damage.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Increase the damage by 1d4 and the persistent damage on a critical hit by 1d4.</p>\n<p>[[/r (@item.level)d4[persistent,fire]]]{Leveled Persistent Fire Damage}</p>"
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/rhenei-3-4.json:1288]: Description and leveled damage

File: packs/data/pfs-season-4-bestiary.db/rhenei-3-4.json [hunk 15]
```diff
@@ -0,0 +1,2114 @@
...
+        },
+        {
+            "_id": "1EsMXmB9tTK4sElQ",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/rhenei-3-4.json:1865]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/thisnabel-1-2.json [hunk 16]
```diff
@@ -0,0 +1,625 @@
...
+        },
+        {
+            "_id": "rm5sOGU46Lbwed4O",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/thisnabel-1-2.json:319]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/thisnabel-1-2.json [hunk 17]
```diff
@@ -0,0 +1,625 @@
...
+                },
+                "description": {
+                    "value": "<p>Thisnabel deals an extra 1d4 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "category": "precision",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/thisnabel-1-2.json:395]: Full RE?

File: packs/data/pfs-season-4-bestiary.db/thisnabel-3-4.json [hunk 18]
```diff
@@ -0,0 +1,625 @@
...
+        },
+        {
+            "_id": "rm5sOGU46Lbwed4O",
+            "flags": {
+                "core": {
+                    "sourceId": "Actor.K9ClMcqPZ4BdZ7iU.Item.1EsMXmB9tTK4sElQ"
+                }
+            },
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Aldori Perry",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/thisnabel-3-4.json:319]: Aldori Parry

File: packs/data/pfs-season-4-bestiary.db/thisnabel-3-4.json [hunk 19]
```diff
@@ -0,0 +1,625 @@
...
+                },
+                "description": {
+                    "value": "<p>Thisnabel deals an extra 1d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "category": "precision",
```
Comment (simonward) [inline packs/data/pfs-season-4-bestiary.db/thisnabel-3-4.json:395]: Full RE?

File: packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json (hunk 1 above)
Comment (TMunsie) [reply packs/data/pfs-season-4-bestiary.db/determined-rhenei-1-2.json:478]: It is the sorcerer class feature that would modify it. The entry in the quest is written as Spell Name (bludgeoning, water) so the traits should be modified but the rest probably isn't worth changing

File: packs/data/pfs-season-4-bestiary.db/determined-thisnabel-1-2.json (hunk 8 above)
Comment (TMunsie) [reply packs/data/pfs-season-4-bestiary.db/determined-thisnabel-1-2.json:395]: No. It's PFS, so it specifically does not use the full rule element. PFS is RAW, and this says only that it does 1dx damage against flatfooted creatures with no caveat as to weapon type.

File: packs/data/pfs-season-4-bestiary.db/determined-thisnabel-3-4.json (hunk 10 above)
Comment (TMunsie) [reply packs/data/pfs-season-4-bestiary.db/determined-thisnabel-3-4.json:395]: Ibid

File: packs/data/pfs-season-4-bestiary.db/thisnabel-1-2.json (hunk 17 above)
Comment (TMunsie) [reply packs/data/pfs-season-4-bestiary.db/thisnabel-1-2.json:395]: ibid

File: packs/data/pfs-season-4-bestiary.db/thisnabel-3-4.json (hunk 19 above)
Comment (TMunsie) [reply packs/data/pfs-season-4-bestiary.db/thisnabel-3-4.json:395]: ibid

**Review by simonward — APPROVED** [review]

**Pull request merged**